build_tools/travis/install.sh

```diff
@@ -53,11 +53,13 @@ if [[ "$DISTRIB" == "conda" ]]; then
     if [[ "$INSTALL_MKL" == "true" ]]; then
         conda create -n testenv --yes python=$PYTHON_VERSION pip nose \
             numpy=$NUMPY_VERSION scipy=$SCIPY_VERSION numpy scipy \
-            cython=$CYTHON_VERSION libgfortran mkl flake8
+            cython=$CYTHON_VERSION libgfortran mkl flake8 \
+            pandas=$PANDAS_VERSION
```

Comment:
I don't believe that first one works. A more standard way might be to construct a command and store it in a variable, then append to it in a conditional.

Alternatively, use something like `$(if [[ -v $PANDAS_VERSION ]]; then echo pandas=$PANDAS_VERSION; fi)` inline.